options.go

```diff
@@ -193,7 +195,6 @@ type Options struct {
 	Compression    int
 	Zoom           int
 	Crop           bool
-	SmartCrop      bool
```

Comment:
Alright, I did that simply by adding the comment. Could you see my latest changes and provide feedback or hit the [merge] button ?